web/public/mgmt/config.yml

```diff
@@ -1275,7 +1285,7 @@ collections:
         }
       - {
           label: "Description for card in For You",
-          name: "descriptionMd",
+          name: "sidebarCardBodyText",
```

Comment:
Why not just call it `description`? I get the intent, but isn't this a little overly specific? We're always going to be dealing with this in the context of either a markdown file or relevant object, in which case, we'll know that it's related to a sidebar card.